internal/storage/storage_handle_test.go

```diff
@@ -25,79 +25,68 @@ import (
 	"github.com/googlecloudplatform/gcsfuse/v2/internal/storage/storageutil"
 	"github.com/jacobsa/oglematchers"
 	. "github.com/jacobsa/ogletest"
+	"github.com/stretchr/testify/suite"
 )
 
 const invalidBucketName string = "will-not-be-present-in-fake-server"
 const projectID string = "valid-project-id"
 
-func TestStorageHandle(t *testing.T) { RunTests(t) }
-
 type StorageHandleTest struct {
+	suite.Suite
 	fakeStorage FakeStorage
 }
 
-var _ SetUpInterface = &StorageHandleTest{}
-var _ TearDownInterface = &StorageHandleTest{}
-
-func init() { RegisterTestSuite(&StorageHandleTest{}) }
+func TestStorageHandleSuite(t *testing.T) {
+	suite.Run(t, new(StorageHandleTest))
+}
 
-func (t *StorageHandleTest) SetUp(_ *TestInfo) {
+func (suite *StorageHandleTest) SetupSuite() {
 	var err error
-	t.fakeStorage = NewFakeStorage()
+	suite.fakeStorage = NewFakeStorage()
```

Comment:
IMO, it's preferable to rename suite to testSuite or ts or sht as `suite` is also imported package.

Reply:
Done